README.md

```diff
@@ -1 +1,179 @@
\ No newline at end of file
+# Tutorial-lessons-frontend
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [:busts_in_silhouette: Authors](#👥-authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [:question: FAQ](#faq)
+- [:memo: License](#license)
+
+# 📖 [Tutorial-lessons-frontend] <a name="about-project"></a>
+
+[Tutorial-lessons-frontend] is a frontend react app for our Tutorial lessons backend. It shows the features of the app and provides a user with better experience and interface
+
+## [Link to the back-end repo](https://github.com/RicardoEgas/tutorial-lessons-backend.git)
+
+## [Link to the kanbarn board](https://github.com/RicardoEgas/tutorial-lessons-backend/projects/1)
+
+## [Link to the kanbarn board at its initial state](.https://github.com/RicardoEgas/tutorial-lessons-backend/issues/10)
+
+## A Team of 4 Members worked on this project - [See list of authors](#👥-authors))
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a>https://react.org/</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Redux</summary>
+  <ul>
+    <li><a>https://redux.js.org/</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [Render Tutorila lessons lists]
+- [Add Tutorials]
+- [Delete Tutorials]
+- [Reserve Tutorials]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd tutorial-lessons-frontend
+  git clone https://github.com/RicardoEgas/tutorial-lessons-frontend.git
+```
+### Install
+
+Install this project with:
+
+```
+  cd tutorial-lessons-frontend
+  npm install
+```
+```
+  npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x @babel/preset-env@7.x  @babel/preset-react@7.x
+```
+```
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+### Usage
+
+To run the project, execute the following command:
+
+```
+  npm run dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+npm test
```

Comment:
- [ ] This command fails because you have not included frontend tests for your app.  Kindly, either add some tests or remove this section in order to prevent confusion. 👍